src/components/atoms/RecurringTransactionOptions.tsx

```diff
@@ -0,0 +1,48 @@
+import React from 'react';
+
+import Input from './Input';
+
+interface IProps {
```

Comment:
I see we're using different conventions for naming interfaces. I have named mine IComponentNameProps.